### ISSUE TITLE
[Bug]: Some apps do not open for editing

### ISSUE THREAD

**Issue by hyuri** [fabrialberio/PinApp#19]
### Describe the bug
Some apps do not open for editing when I click on them. I click on them and nothing happens. No matter the packaging format, it seems (tested on AppImages and Flatpaks).

When I click on the apps that don't open for editing, I get this error when running the app through the command line (flatpak run io.github.fabrialberio.pinapp):
```
Traceback (most recent call last):
  File "/app/share/PinApp/pinapp/window.py", line 53, in <lambda>
    self.pins_view.connect('file-open', lambda _, f: self.open_file(f))
  File "/app/share/PinApp/pinapp/window.py", line 124, in open_file
    self.file_page.load_file(file)
  File "/app/share/PinApp/pinapp/file_page.py", line 207, in load_file
    self.update_file()
  File "/app/share/PinApp/pinapp/file_page.py", line 215, in update_file
    self._update_locale()
  File "/app/share/PinApp/pinapp/file_page.py", line 319, in _update_locale
    str(LocaleString.current().closest(
  File "/app/share/PinApp/pinapp/desktop_entry.py", line 31, in current
    return cls.parse(getlocale(category)[0])
  File "/usr/lib/python3.10/locale.py", line 602, in getlocale
    raise TypeError('category LC_ALL is not supported')
TypeError: category LC_ALL is not supported
```

The app also starts with this error:
```
(process:2): Gtk-WARNING **: 06:35:40.814: Locale not supported by C library.
	Using the fallback 'C' locale.
```

My Locale is a mix of en_US language and pt_BR (Brasil) date, time and other formats.

 ### To Reproduce
1. Go to "Your pins"
2. Scroll down to an app or search for it using the search bar (Ctrl + F)
3. Click on the app
4. See how the app does not open for editing

### Screenshots
Clicking on the app highlights it but does not open it for editing.
![image](https://user-images.githubusercontent.com/18584014/210331296-ca39b1af-7fbe-47fc-a576-6844eb55f7cc.png)

### Linux distribution
Pop!_OS 21.10
Gnome 40.4.0 (on X11)

**Comment by muznyo** [comment]
I still have this issue with some apps

```
Traceback (most recent call last):
  File "/app/share/PinApp/pinapp/window.py", line 55, in <lambda>
    self.installed_view.connect('file-open', lambda _, f: self.open_file(f))
  File "/app/share/PinApp/pinapp/window.py", line 124, in open_file
    self.file_page.load_file(file)
  File "/app/share/PinApp/pinapp/file_page.py", line 207, in load_file
    self.update_file()
  File "/app/share/PinApp/pinapp/file_page.py", line 215, in update_file
    self._update_locale()
  File "/app/share/PinApp/pinapp/file_page.py", line 319, in _update_locale
    str(LocaleString.current().closest(
  File "/app/share/PinApp/pinapp/desktop_entry.py", line 31, in current
    return cls.parse(getlocale(category)[0])
  File "/usr/lib/python3.10/locale.py", line 602, in getlocale
    raise TypeError('category LC_ALL is not supported')
TypeError: category LC_ALL is not supported
```

**Comment by fabrialberio** [comment]
This bug has been fixed with #20.
It still happens because there hasn't yet been a release since the pull request has been merged.
